src/scenes/bootScene.ts

```diff
@@ -2,86 +2,125 @@ import { getGameWidth, getGameHeight } from '../helpers';
 import { getGameState } from '../utilities/localStorage';
```

Comment:
this file didn't match the prettierrc so vscode wanted to rewrite all of it

also gotta keep that LOC up or else i'd get 🪓